docs/en/integrations/build-integration.asciidoc

```diff
@@ -208,20 +208,55 @@ apache
 │   │   └───elasticsearch/ingest_pipeline
 │   │   │      default.yml
 │   │   └───fields
-│   │          agent.yml <1>
-│   │          base-fields.yml <2>
-│   │          ecs.yml <3>
-│   │          fields.yml <4>
+│   │          agent.yml
+│   │          base-fields.yml
+│   │          ecs.yml
+│   │          fields.yml
```

Comment:
We should mention somewhere that these names are a convention, we don't really do any special handling depending on their names and any file name with `.yml` extension would work. One could define all their fields without any order in `myfields.yml` and it would work.